src/routes/Introduction/components/Slide.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+
+import React, { Component } from 'react'
+
+import style from '../introduction.m.scss'
+
+type Props = {|
+|}
+
+type State = {|
+  animationData: ?string,
+|}
+
+export class Slide extends Component<Props, State> {
+  constructor(props: Props) {
+    super(props)
+
+    this.lottie = null
+    this.canvasRef = React.createRef()
+
+    this.state = {
+      animationData: null,
+    }
+
+    Promise
+      .all([
+        import('lottie-web'),
+        import('../animations/vault'),
+      ])
```

Comment:
we need to show loader while they are fetching

Reply:
Actually, there is another solution: we will show the first frame of animation as static image (packed with the app like every other image), then download animation and display it instead of the image. So it will look like static image comes to life (and no loader needed). 

Design team agrees it is a decent solution. What do you think?

Reply:
I think it is great idea 👍